chat/tests/int/api-gateway.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, it, expect } from 'vitest';
+import { testConfig } from '../common/config';
+
+describe('api gateway', () => {
```

Comment:
To align on some of the conventions we discussed, should we move this to the `acc` folder?